frontend/src/components/Profile/MyFriends.vue

```diff
@@ -166,6 +167,11 @@ export default defineComponent({
       await this.$store.dispatch('spectateUser', userId);
       this.$router.push('/game');
     },
+    async handleChallenge(userId: number) {
+      console.log("spectating user " + userId);
```

Comment:
```suggestion
      console.log("spectating user " + userId);
```
If this comment isn't necessary anymore

Reply:
done